ci-operator/SECRETS.md

```diff
@@ -18,15 +18,26 @@ vet that interaction for correctness.
 ### Aggregate IaaS Secrets for use in Cluster Tests
 
 A set of secrets exists that contain aggregated information for ease of mounting
-into tests that are interacting with an IaaS hosted by a cloud. The following secrets
-currently exist:
+into tests that are interacting with an IaaS hosted by a cloud.
+
+All secrets in this category have a `pull-secret` key containing:
+
+- credentials for pulling OpenShift images from Quay
+- credentials for authenticating to telemetry (retrieved from
+  [try.openshift.com](https://try.openshift.com) under the
+  ccoleman+openshift-ci-test@redhat.com account)
+- the service account token from the `ocp` namespace in each CI cluster, added
+  with `oc registry login --to=/tmp/pull-secret -z default -n ocp`, allowing
+  images to be pulled as `system:authenticated`
+
+The following secrets currently exist:
 
 #### `cluster-secrets-aws`
 
 |       Key        | Description |
 | ---------------- | ----------- |
 | `.awscred`       | Credentials for the AWS EC2 API. See the [upstream credentials doc](https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html). |
-| `pull-secret`    | Credentials for pulling OpenShift images from Quay and for authenticating to telemetry. Retrieved from [try.openshift.com](https://try.openshift.com) under the ccoleman+openshift-ci-test@redhat.com account, and has the service account token from the `ocp` namespace added with `oc registry login --to=/tmp/pull-secret -z default -n ocp`.|
+| `pull-secret`    | See above. |
```

Comment:
Could we give the "above" a heading and link to it from the "See above" sentences? Will be useful especially for the ones at the bottom.

Reply:
@bbguimaraes ?

Reply:
Updated.